lib/fabrik/OobBackupManager.js

```diff
@@ -106,7 +106,8 @@ class OobBackupManager {
                     root_folder: CONST.FABRIK_OUT_OF_BAND_DEPLOYMENTS.ROOT_FOLDER_NAME
                   }, {
                     state: lastOperation.state,
-                    logs: logs
+                    logs: logs,
+                    snapshotId: lastOperation.snapshotId
```

Comment:
guess it would still work when deleting because snapshotId would be coming as undefined, but still it would be a good idea to add this attrib only when the snapshotId is present.

Reply:
We are doing JSON.sringify() which discards parameter having value undefined.  
Here: https://github.com/cloudfoundry-incubator/service-fabrik-broker/blob/master/lib/iaas/CloudProviderClient.js#L147 

Also we are doing same here
https://github.com/cloudfoundry-incubator/service-fabrik-broker/blob/master/lib/fabrik/DirectorManager.js#L687

In metadata snapshotId doesn't come when it is undefined.

Reply:
Created a ticket https://github.com/cloudfoundry-incubator/service-fabrik-broker/issues/124